doc/get_started.rst

```diff
@@ -3,110 +3,89 @@
--------
-
-This package can be installed through `pip`.  In order to allow benchopt to automatically
-install solvers dependencies, the install needs to be done in a `conda` environment.
+The recommended way to use benchopt is within a conda environment to fully-benefit from all its features.
+Hence, start by creating a dedicated conda environment and then activate it.
 
 .. prompt:: bash $
 
     conda create -n benchopt python
     conda activate benchopt
 
-To get the **latest release**, use:
+Benchopt is available on PyPi. You can get the **stable version** via ``pip`` by running the following command
 
 .. prompt:: bash $
 
-    pip install benchopt
+    pip install -U benchopt
 
-And to get the **latest development version**, you can use:
+Eager to try out the **development version**? you can run instead
 
 .. prompt:: bash $
 
-    pip install -U -i https://test.pypi.org/simple/ benchopt
-
-This will install the command line tool to run the benchmark. Then, existing
-benchmarks can be retrieved from GitHub or created locally. To discover which
-benchmarks are presently available look for
-`benchmark_* repositories on GitHub <https://github.com/benchopt/>`_,
-such as for `Lasso -- l1-regularized linear regression <https://github.com/benchopt/benchmark_lasso>`_.
-This benchmark can be retrieved locally with:
+    pip install -U -i https://test.pypi.org/simple/benchopt
 
-.. prompt:: bash $
+.. attention::
 
-    git clone https://github.com/benchopt/benchmark_lasso.git
+   The **development version** is a work in progress and hence might contain incomplete features.
+   A typical user is advised to use the **stable version** instead.
 
+With benchopt being installed, you get access to the :ref:`Command Line Interface (CLI) <cli_ref>`,
+which enables simple and easy manipulation of benchmarks just from the terminal.
 
 
-Run existing benchmark
-----------------------
+Run an existing benchmark
+-------------------------
 
-This section illustrates benchopt's command line interface on the `Lasso benchmark <https://github.com/benchopt/benchmark_lasso>`_; the syntax is applicable to any benchmark.
-All this section assumes that you are in the parent folder of the ``benchmark_lasso`` folder.
-The ``--env`` flag specifies that everything is run in the ``benchopt_benchmark_lasso`` ``conda`` environment.
-
-**Installing benchmark dependencies**: benchopt exposes a CLI to install solvers' dependencies automatically.
-It only works inside a ``conda`` environment. To install all requirements of the benchmark, make sure a ``conda``
-environment is activated and run:
-
-.. prompt:: bash $
+Let's get the first steps with benchopt by comparing some solvers of
+`Lasso problem <https://en.wikipedia.org/wiki/Lasso_(statistics)>`_ on a
+`Leukemia dataset <https://www.science.org/doi/10.1126/science.286.5439.531>`_.
 
-    benchopt install --env ./benchmark_lasso
+Benchopt community maintains :ref:`several optimization benchmarks <available_benchmarks>`
```

Comment:
IMO this information can be ommitted here, documentation should go straight to the point.

Reply:
I think it prepares well the floor for using Benchmark Lasso?
Also, It points to the ecosystem of benchopt benchmark, which IMO valuable for the visibility of this benchmarks reops.

Reply:
yesterday's experience with Nelly hinted that there was too much information on the page, which make critical information harder to find. I'll remove this part
The other benchmarks are exposed on the repo's readme